index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <img src="/images/Welcome.png" alt="welcome button at the top left">
+        <img src="/images/Union.png" alt="a union of three vectors">
+    </header>
+    <br>
+    <div class="second">
+      <h1>  Hey there. I’m Badru
+I’m a software developer</h1>
+    <p>
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
```

Comment:
- [ ] you should do the same here and try to add the style provided in the Figma design in order to match the design 
- Figma
<img width="352" alt="Screen Shot 2022-09-21 at 18 27 14" src="https://user-images.githubusercontent.com/61790587/191571431-9b8c74cb-9eb0-4af5-b53f-5ee3cb7bc900.png">
<img width="216" alt="Screen Shot 2022-09-21 at 18 27 27" src="https://user-images.githubusercontent.com/61790587/191571447-0320b2e2-f929-4ef8-a55e-d52d1e2b8fa9.png">
- Yours 
<img width="312" alt="Screen Shot 2022-09-21 at 18 27 38" src="https://user-images.githubusercontent.com/61790587/191571482-61c56e60-c7de-49e6-b364-991eb5b1bd0e.png">